client/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release Notes
 
```

Comment:
```suggestion

## 1.X.X (YYYY-MM-DD)

…

```

Reply:
This doesn't seem to be a good idea since the change will be done on `master`.

Reply:
Crap. I'm used to Yseop Suite, where the master→develop merge is made through an intermediate branch. Sorry.